rsts/concepts/flyte_console.rst

```diff
@@ -0,0 +1,202 @@
+    :alt: Versioned executions
+
+    Versioned executions are viewable in the UI.
+
+|
+
+Trigger Versioned Executions
+----------------------------
+
+Every registered workflow is versioned and can be triggered anytime.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/trigger_versioned_executions.png
+    :alt: Trigger versioned execution
+
+    Trigger versioned executions.
+
+|
+
+Inspect Executions
+------------------
+
+Executions can be inspected through the UI. Inputs and Outputs can be viewed for every node and execution.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/inspect_execution_001.png
+    :alt: Node's inputs and outputs
+
+    Every execution node's inputs and outputs are viewable.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/inspect_execution_002.png
+    :alt: Execution's inputs and outputs
+
+    Every execution's inputs and outputs are viewable.
+
+|
+
+Logs should be accessible as well.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/inspect_execution_003.png
+    :alt: Logs
+
+    Kubernetes logs are viewable.
+
+|
+
+Every execution has two views: Node and Graph.
+
+A node will encapsulate an instance of a task, but it can also contain an entire subworkflow or trigger a child workflow.
+More about nodes can be found in :std:ref:`divedeep-nodes`.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/inspect_execution_004.png
+    :alt: Nodes
+
+    Inspect execution's nodes in the UI.
+
+|
+
+Graph view showcases the DAG.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/inspect_execution_005.png
+    :alt: DAG
+
+    Inspect execution's DAG in the UI.
+
+|
+
+Clone Execution
+----------------
+
+An execution in the RUNNING state can be cloned.
+
+Click on the kebab menu on the top right corner of the UI.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/clone_execution_001.png
+    :alt: Clone execution
+
+    Step 1: Click on the kebab menu.
+
+|
+
+Click on the **Clone Execution** button.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/clone_execution_002.png
+    :alt: Clone execution
+
+    Step 2: Clone execution.
+
+|
+
+Relaunch Execution
+------------------
+
+**Relaunch** button allows you to relaunch an execution with pre-populated inputs.
+
+|
+
+.. figure:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flyteconsole/relaunch_execution.png
+    :alt: Relaunch execution
+
+    Relaunch an execution.
+
+|
+
+A pop-up window gets displayed and it allows you to modify the version and inputs.
+
```

Comment:
Maybe 'A pop-up window appears that allows user to modify the version and inputs'?